src/General/Heading/Heading.test.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+
+import * as renderer from 'react-test-renderer';
+import '@testing-library/jest-dom/extend-expect';
+import { render } from '@testing-library/react';
+
+import Heading from './Heading';
+
+const heading = 'This is a Heading';
+
+it(`<Heading> should match snapshot when the component is called`, () => {
+  const HeadingSnapshot = renderer
+    .create(<Heading>{heading}</Heading>)
+    .toJSON();
+  expect(HeadingSnapshot).toMatchSnapshot();
+});
+
+it('<Heading> should display the correct text for the heading when rendered', () => {
+  const { queryByText } = render(<Heading>{heading}</Heading>);
+  const HeadingContent = queryByText(heading);
+  expect(HeadingContent).toBeVisible();
```

Comment:
@ritikamotwani this test is actually covered by the snapshot test so don't need this.